### ISSUE TITLE
[Header] implement a custom header builder, similar to the custom footer functionality 

### ISSUE THREAD

**Issue by vatoyiit** [KriesiMedia/Enfold-Feature-Requests#31]
> Something I found very useful in Divi by Elegant Themes. They include a header element builder. Simillar to the way the Enfold Footer Builder works. This would involve having a range of new ALB elements covering things like Site Logo, Menu Bar, Social Icon Block, WooCommerce Trolly Block, Search Icon Block and mobile rules available for these elements.

https://kriesi.at/support/topic/notes-for-dev-feature-requests/

**Comment by webumy** [comment]
Definitely! It would be great to edit the header and footer unlimitedly, like the "essentials theme" element.
